### DISCUSSION TITLE
May I ask how to handle Chinese polyphonic characters?

### DISCUSSION THREAD

**Discussion by syf2012** [coqui-ai/TTS#3444] [General Q&A]
When using Chinese language, there may be polyphonic processing errors. How can I config it?

**Comment by aaron-lii** [comment]
你指的是多音字的处理吗？
https://github.com/coqui-ai/TTS/blob/55c7063724ec28085f9a61210e3dd3e34753c4d0/TTS/tts/layers/xtts/tokenizer.py#L653
We can manually change the pinyin in the ‘txt’ variable before this line of code.

**Reply by syf2012** [reply]
If it supports SSML, it would be even better because manual code processing is not very flexible, but your idea also points the way
